CMSgov/mint-app
# [EASI-2816] Search Changes in Model Plan Hierarchy by ID

# EASI-2816

This query will fetch all changes related to a model plan by searching with a model plan ID.
It supports:
- Date range
- Returns results in reverse chronological order
- Paging (via `limit` and `offset` search parameters)

## Changes and Description

- Added a new endpoint named SearchChangeTableByModelPlanIDAndDateRange



## How to test this change

Run the endpoint SearchChangeTableByModelPlanIDAndDateRange. Verify that it functions and returns the proper data.



## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
